File: Array/Javascript/reverse_array.js
const reverseArray = (arr) => {
    let reverseArray = []
    for (let i = arr.length - 1; i >= 0; i--) {
        reverseArray.push(arr[i])
    }
    return reverseArray
}

const array = [1, 2, 3, 4, 5, 7]
console.log(reverseArray(array))

File: Array/Go/reverse_array.go
package main

import "fmt"

func reverseArray(arr string) string {
	var output string = ""
	for i := len(arr) - 1; i >= 0; i-- {
		fmt.Println("arr====+", arr[i])
		output = "a"
	}
	return output
}

func main() {
	fmt.Println(reverseArray("abc"))
}
